docs/build-tools/maven.md

```diff
@@ -6,13 +6,17 @@ sidebar_label: Maven
 
 ## Getting Started
 
-Configuring bloop with Maven projects speeds up developer workflows and gives you access to a modern Scala toolchain, despite Maven traditionally lacking good Scala suppport.
+Configuring bloop with Maven projects speeds up developer workflows and gives
```

Comment:
Not really related to this PR, but I noticed that Bloop is often written `bloop` with no capital B.

```suggestion
Configuring Bloop with Maven projects speeds up developer workflows and gives
```